client/signup/steps/site-info-collection/form-components.tsx

```diff
@@ -0,0 +1,209 @@
+import styled from '@emotion/styled';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { ChangeEvent } from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormInputValidation from 'calypso/components/forms/form-input-validation';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import FormTextArea from 'calypso/components/forms/form-textarea';
+import SocialLogo from 'calypso/components/social-logo';
+
+const Label = styled( FormLabel )`
+	color: var( --studio-gray-50 );
+	font-weight: 400;
+	margin-bottom: 24px;
+`;
+
+const TextInput = styled( FormTextInput )`
+	//todo change this
+	border-radius: 5px !important;
+	font-size: 13px !important;
+	padding: 12px 16px;
+`;
+
+const TextArea = styled( FormTextArea )`
+	border-radius: 5px !important;
+	font-size: 13px !important;
+	padding: 12px 16px;
+`;
```

Comment:
Wow do we really need all these `importants`? Feels like we might doing something wrong. Are you sure we need to make the design pixel perfect ? 

@onuro Have we decided to move away from all fundamental numbers like border-radius and font-size in the existing WordPress design language. The last time I tried to make a design pixel perfect, the advice I got from @jsnajdr  was to move cautiously and stick to the existing design language wherever possible.

But since the latest blogger experience design language is a bit different I am not sure now. Maybe @p-jackson can weigh in a bit here? Did you reuse existing WordPress components and override fundamental design parameters to be pixel perfect?

@aneeshd16  In any case maybe we can rely on specificity to override this css.
```suggestion
// && will simply copy the classname twice
const TextArea = styled( FormTextArea )`
	padding: 12px 16px;
        '&&': {
	    border-radius: 5px;
	    font-size: 13px;
        }
`;
```

Reply:
> Wow do we really need all these important?

Not at all! Good catch! I had added a TODO comment to this file: https://github.com/Automattic/wp-calypso/blob/b0f301bcf769279652629aa9f00bf3fefc4530cf/client/signup/steps/site-info-collection/form-components.tsx#L18
...but I missed changing it.

Reply:
> In any case maybe we can rely on specificity to override this css.

@jdc91, can you please check the suggested change? It gives me a TS error and does not apply the styles:
![image](https://user-images.githubusercontent.com/5436027/150090296-8f88a485-fca5-4a6e-83b0-36e3e79684dd.png)

Reply:
This works for me:
```
input&.form-text-input {
		border-radius: 4px;
		font-size: 13px;
		padding: 12px 16px;
		&:focus,
		&:focus:hover {
			border-color: #646970;
			box-shadow: 0 0 0 2px #e2eaf1;
		}
	}
```